src/PKSim.Core/Snapshots/Mappers/CompoundPropertiesMapper.cs

```diff
@@ -66,7 +66,7 @@ private ProtocolSelection protocolPropertiesFrom(ProtocolProperties protocolProp
 
       private Task<CompoundProcessSelection[]> snapshotProcessSelectionFrom(CompoundProcessesSelection compoundProcessesSelection)
       {
-         return _processMappingMapper.MapToSnapshots(compoundProcessesSelection.AllEnabledProcesses());
+         return _processMappingMapper.MapToSnapshots(compoundProcessesSelection.AllProcesses());
```

Comment:
Here we were only mapping the one activated. The SimulationConfiguration adds the missins enzymes by default so this was creating the mismatch on configure